sdk/src/main/java/io/radar/sdk/RadarTrackingOptions.kt

```diff
@@ -277,7 +277,13 @@ data class RadarTrackingOptions(
         /**
          * Determines the id of the notification. Optional, defaults to `20160525`.
          */
-        val id: Int? = null
+        val id: Int? = null,
+
+        /**
+         * Determines the user-facing channel name, which can be viewed in notification settings for the application.
+         * Optional, defaults to `"Location Tracking"`.
+         */
+        val channelName: String? = null
```

Comment:
I would say this doesn't belong in tracking options, as it's device-specific and client-configured, so it wouldn't make sense to control this from the server-side as it's too tightly configured to the client implementation.

Can this live with the other foreground options on local storage?

Reply:
ah, okay I see that now. yes, so this seems fine then given that it won't conflict with the server tracking options